design-docs/tooling-api-parameterized-custom-models.md

```diff
@@ -0,0 +1,164 @@
+
+# Feature: Gradle plugin builds custom tooling models using parameter passed by client
```

Comment:
You can remove this doc, we already have the Google Doc and we recently deleted the design-docs folder, because they just go out of date over time anyway :)

Reply:
Cool! Done.